test/endtoend/serviceEndToEnd.test.ts

```diff
@@ -29,7 +30,7 @@ const nextConfig = {
     watcherResponseConfirmations: 0
 };
 
-const provider = new ethers.providers.Web3Provider(ganache);
+const provider = new ethers.providers.Web3Provider(ganache as ethers.providers.AsyncSendable);
```

Comment:
Same here, probably better to cast at the declaration - I looked into the source code and it seems they;'re missing some types. It looks like the types they're referencing are web3 0.x types whereas the actual included packages are web3 1.x. So maybe they updated the js, but forgot to do the types? Hopefully they'll fix it in the future.